nuxeo-jsf/nuxeo-platform-forms-layout-client/src/main/java/org/nuxeo/ecm/platform/forms/layout/facelets/plugins/AbstractDirectorySelectWidgetTypeHandler.java

```diff
@@ -42,6 +46,11 @@
         directoryName, displayAll, displayObsoleteEntries, filter, localize, dbl10n;
     }
 
+    /**
+     * @since 9.1
+     */
+    public static final String ATTRIBUTE_STYLE_CLASS = "styleClass";
+
```

Comment:
This does not need to be public, and should actually be a list, we can review the main attributes that will cause the generation of a span tag (probably most of view properties of the corresponding widget). In fact this list can directly be hardcoded in the method below.